widgetry/shaders/vertex_webgl1.glsl

```diff
@@ -0,0 +1,55 @@
+#version 100
+
+precision mediump float;
+precision mediump sampler2D;
+
+// (x offset, y offset, zoom)
+uniform vec3 transform;
+// (window width, window height, z value)
+uniform vec3 window;
+
+// in
+attribute vec3 position;
+attribute vec4 color;
+attribute float texture_index;
```

Comment:
Unlike the fragment shader, the vertex shader for webgl 1.0 is almost identical to the WebGL 2.0 vertex shader. The only difference is in how variables/uniforms are declared.